ansible/roles/kafka/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+---
+
+- name: create kafka group
+  group: name={{ kafka_group }} system=yes state=present
+
+- name: add user "kafka"
+  user: name={{ kafka_group }} group={{ kafka_group }} shell=/sbin/nologin system=yes state=present
+
+- name: Download Kafka
+  get_url: url="http://{{ apache_mirror }}/kafka/{{ kafka_version }}/kafka_2.10-{{ kafka_version }}.tgz" dest=/opt/kafka_2.10-{{ kafka_version }}.tgz
```

Comment:
yeah is a bit weird. is the scala version (which we aren't going to use/care about). the [downloads page](http://kafka.apache.org/downloads.html) recommended 2.10 if you don't care.